src/main/java/org/example/util/TransformerDtoApartment.java

```diff
@@ -0,0 +1,36 @@
+package org.example.util;
+
+import org.example.dto.ApartmentDTO;
+import org.example.entity.Apartment;
+import org.example.repository.RealtyAgentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TransformerDtoApartment {
+
+    @Autowired
+    protected RealtyAgentRepository realtyAgentRepository;
+
+    public Apartment createTransformDtoApartment(int id, ApartmentDTO apartmentDTO) {
```

Comment:
Transformer shouldn't devide methods into create*** and edit***.
It should just transform from DTO to bean and vise versa.
That's mean, that you should have ideally just 2 methods:

```
public Apartment populateBeanFromDTO(Apartment apartment, ApartmentDTO apartmentDTO)
```

and

```
public Apartment populateDTOFromBean(ApartmentDTO apartmentDTO, Apartment apartment)
```